### PR TITLE
MLI BYORs Developer Guide + explainer examples

### PR DESCRIPTION
**Pull request by dvorka**
This PR brings MLI BYORs Developer Guide and explainer examples.

### CODE REVIEW

File: explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ When MLI user starts interpretation, model compatible explainers (from the avail
 ### What is the role of recipes?
```
Comment (navdeep-G) [inline explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md:42]: Is all of this suppose to go into the DAI user guide? If so, then it should be `*.rst` files. Right @zainhaq-h2o @NidhiMehta ?

**Review by navdeep-G — APPROVED** [review]

File: explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md (hunk 1 above)
Comment (zainhaq-h2o) [reply explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md:42]: @navdeep-G I can definitely add this info to the DAI user guide if the MLI team thinks it's a good idea to have it included there.

Comment (dvorka) [reply explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md:42]: @navdeep-G yes, I think that it's desired to make this document part of DAI documentation.

@zainhaq-h2o @NidhiMehta will you convert it to `*.rst` or is it expected that I will do it myself. Please let me know.

Comment (zainhaq-h2o) [reply explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md:42]: @dvorka I will convert it for you. I will create a PR and add you as a reviewer.

Comment (dvorka) [reply explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md:42]: Thank you Zain - I really appreciate it!

**Pull request merged**